### PR TITLE
[JetBrains] Replace "Status" by "JetBrains Launcher" and run it in a separate image from IDEs

### PR DESCRIPTION
**Pull request by andreafalzetti**
## Description
This PR extracts the `status` program from the JB IDE images. We're adding a new image called `jb-launcher` and it will make iterating on this logic much faster, avoiding to re-build the images each time.

## Related Issue(s)

Related to #6740 

## How to test

1. Open the Preview Environment from this PR: https://ide-jetbra3bd70cdaa4.preview.gitpod-dev.com/workspaces
2. For Stable and Latest IntelliJ IDEA, confirm if:
    1. Prebuilds are working as usual using this branch: https://github.com/andreafalzetti/gitpod-prebuilds/tree/jb-3 (You can [join this Team](https://ide-jetbra3bd70cdaa4.preview.gitpod-dev.com/teams/join?inviteId=7b597208-d1c7-4d4d-9271-e42a14830e59) in the Preview Environment)
    2. The IDE connects to the workspace and has Gitpod Plugin installed.

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [x] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Comment by roboquat** [comment]
@andreafalzetti: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: components/ide-service/example-ide-config.json [hunk 1]
```diff
@@ -72,7 +96,7 @@
           "phpstorm"
         ],
         "installationSteps": [
-          "If you don't see an open dialog in your browser, make sure you have the \u003ca target='_blank' class='gp-link' href='https://www.gitpod.io/docs/ides-and-editors/jetbrains-gateway#getting-started-jetbrains-gateway'\u003eJetBrains Gateway with Gitpod Plugin\u003c/a\u003e installed on your machine, and then click \u003cb\u003e${OPEN_LINK_LABEL}\u003c/b\u003e below."
+          "If you don't see an open dialog in your browser, make sure you hlave the \u003ca target='_blank' class='gp-link' href='https://www.gitpod.io/docs/ides-and-editors/jetbrains-gateway#getting-started-jetbrains-gateway'\u003eJetBrains Gateway with Gitpod Plugin\u003c/a\u003e installed on your machine, and then click \u003cb\u003e${OPEN_LINK_LABEL}\u003c/b\u003e below."
```
Comment (felladrin) [inline components/ide-service/example-ide-config.json:99]: Typo.

Comment (felladrin) [reply components/ide-service/example-ide-config.json]: Resolved.

**Comment by andreafalzetti** [comment]
We've tested regular workspaces start fine, also warmup logs are as expected, we're going to merge this one cc @akosyakov

**Review by felladrin — APPROVED** [review]
Working great! ✅ 

Tested on IntelliJ IDEA Stable:
image

image
image

File: components/ide/jetbrains/image/status/hot-swap.sh [hunk 2]
```diff
@@ -1,45 +0,0 @@
-#!/bin/bash
```
Comment (akosyakov) [inline components/ide/jetbrains/image/status/hot-swap.sh:1]: this one we should keep but fix later cc @andreafalzetti @felladrin

Comment (felladrin) [reply components/ide/jetbrains/image/status/hot-swap.sh:1]: You mean `components/ide/jetbrains/launcher/hot-deploy.sh` [[1](https://github.com/gitpod-io/gitpod/pull/15308/files/b69c1dff1608681330e8d2b6332ae7789d4442a4#diff-976f1dfa9e5c722d2b812509cebfe510cec3dde67e89ad3aa10e4045e674d77e)]? If so, we kept it.

Comment (akosyakov) [reply components/ide/jetbrains/image/status/hot-swap.sh:1]: no, not-swap as well, it allows to iterate on jb-launcher within a running workspace

**Pull request merged**